riak_test/tests/gc_tests.erl

```diff
@@ -206,3 +223,47 @@ verify_riak_object_remaining_for_bad_key(RiakNodes, GCKey, {{Bucket, Key}, UUID}
     lager:info("As expected, BAD manifest in GC bucket remains,"
                " stand off orphan manfiests/blocks: ~p", [Manifest]),
     ok.
+
+verify_partial_gc_run(CSNode, RiakNodes,
+                      {MegaSec0, Sec0, _},
+                      {MegaSec1, Sec1, _}) ->
+    Start0 = MegaSec0 * 1000000 + Sec0,
+    End0 = MegaSec1 * 1000000 + Sec1,
+    Interval = erlang:max(1, (End0 - Start0) div 3),
+    Starts = [ {Start0 + N * Interval, Start0 + (N+1) * Interval}
+               || N <- lists:seq(0, 3) ],
```

Comment:
Ugh, in the above "alternative" , last {start, end} may be coincide (for example start=0 and end=6). Then gc will fail... :cry:  Sorry for wrong example :see_no_evil: